src/binary/buffer.rs

```diff
@@ -0,0 +1,90 @@
+use crate::common::{Bytes32, Position};
+use core::mem::size_of;
+
+const BUFFER_SIZE: usize = size_of::<Position>() + size_of::<Bytes32>();
+pub type Buffer = [u8; BUFFER_SIZE];
+pub const DEFAULT_BUFFER: &Buffer = &[0; BUFFER_SIZE];
+
+struct Schema {}
```

Comment:
What is the purpose of this local struct that's never instantiated? Couldn't we simply replace the const fn's with const vars?

Reply:
There isn't any material purpose that can't be achieved with constants. I'll make that change now.